Userland/Applications/HexEditor/ValueInspectorModel.h

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright (c) 2022, Timothy Slater <tslater2006@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Hex.h>
+#include <AK/NonnullRefPtr.h>
+#include <AK/String.h>
+#include <AK/Utf8View.h>
+#include <AK/Vector.h>
+#include <LibGUI/Model.h>
+
+class ValueInspectorModel final : public GUI::Model {
+public:
+    enum ValueType {
+        SignedByte,
+        UnsignedByte,
+        SignedShort,
+        UnsignedShort,
+        SignedInt,
+        UnsignedInt,
+        SignedLong,
+        UnsignedLong,
+        Float,
+        Double,
+        __Count
+    };
+
+    enum Column {
+        Type,
+        Value
+    };
+
+    explicit ValueInspectorModel()
+    {
+        for (int i = 0; i < ValueType::__Count; i++) {
+            set_parsed_value(static_cast<ValueType>(i), "");
+        }
```

Comment:
Drop the braces for single-line enclosures like this one.

Reply:
Fixed!